esmvaltool/recipes/recipe_sea_surface_salinity.yml

```diff
@@ -0,0 +1,62 @@
+---
+documentation:
+  description: |
+    Recipe for evaluating models against ESACCI Sea Surface Salinity dataset
+
+  authors:
+    - vegas-regidor_javier
+
+  maintainer:
+    - vegas-regidor_javier
+
+  references:
+    - contact_authors
+
+  projects:
+    - cmug
+
+preprocessors:
+  timeseries:
+    extract_shape:
+      shapefile: /home/jvegas/World_Seas_IHO_v3/World_Seas_IHO_v3.shp
```

Comment:
I think instead of specifying the complete path here, it would be better to follow e.g. the example of recipe_shapeselect.yml and use auxiliary_data_dir and specify the filename of the shapefile in the recipe only.